docs/general/ddl/constraints.rst

```diff
@@ -65,9 +65,40 @@ Example::
     ... );
     CREATE OK, 1 row affected (... sec)
 
+.. NOTE::
+
+   For further details see :ref:`not_null_constraint`.
+
+Check
+=====
+
+The check constraint can be used on any column, it prevents inserts and
+updates of values that do not pass the check test.
+
+Example::
+
+    cr> create table my_table3 (
+    ...   first_column integer primary key,
+    ...   second_column integer check(second_column >= 0),
+    ...   third_column string constraint not_you check(third_column <> 'me'),
+    ...   check(first_column >= 100),
+    ...   constraint is_even check(second_column % 2 = 0)
+    ... );
+    CREATE OK, 1 row affected  (... sec)
```

Comment:
```suggestion
A check constraint allows you to specify that the values in a certain column
must satisfy a boolean expression. This can be used to ensure data integrity. For example: If you have a table to store metrics from sensors and you want to ensure that negative values are rejected::


    cr> create table metrics (
    ...   id TEXT PRIMARY KEY,
    ...   weight double CHECK (weight >= 0)
    ... );
    CREATE OK, 1 row affected  (... sec)
```

I think for the teasing example it is better to keep it simple - only one
constraint of many. The other document then elaborates on the different
variants.

I also think we should try to stop using `my_table...`, but try to come up with a IoT or other real-world examples.